lib/hope-tab.js

```diff
@@ -243,14 +245,6 @@ window.HopeTab = fxosComponent.register('hope-tab', {
         box-sizing: content-box;
       }
 
-      .url .field .reader {
-        background-image: url(assets/urlbar_reader_mode_icon.png);
-      }
-
-      .url .field .refresh {
-        background-image: url(assets/urlbar_refresh_icon.png);
```

Comment:
Good plan the reader one certainly looks much better. Not sure what to do for the home icon, it was just a circle before but neither the new one I used right.